.github/workflows/integration-test.yml

```
@@ -1354,4 +1358,99 @@ jobs:
     permissions:
       id-token: write
       contents: read
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
+
+
+  GPUEndToEndTest:
```

Rename this to match the matrix naming.
```suggestion
  EKSAddonTest:
```